Project.toml

```diff
@@ -28,16 +28,11 @@ DataStructures = "0.18"
 ForwardDiff = "0.5, 0.6, 0.7, 0.8, 0.9, 0.10"
 JSON = "~0.21"
 JSONSchema = "1"
-LinearAlgebra = "1.6"
 MutableArithmetics = "1"
 NaNMath = "0.3, 1"
 OrderedCollections = "1"
 PrecompileTools = "1"
-Printf = "1.6"
-SparseArrays = "1.6"
 SpecialFunctions = "0.8, 1, 2"
-Test = "1.6"
-Unicode = "1.6"
```

Comment:
I've reverted these changes for now because of https://github.com/jump-dev/SCS.jl/pull/284